apps/frontend/src/lib/imports/imports.ts

```diff
@@ -175,7 +176,8 @@ export async function refreshData(): Promise<void> {
 
         if (accessToken.expires_at && now > expiresAt) {
           // TODO: swap to use refresh token
-          toast.error(`${app.toString()} token has expired, go through OAuth flow again`);
+          storage.deleteOAuthAccessToken(option.type);
+          toast.error(`${capitalized} token has expired, go through OAuth flow again.`, {duration: 5000});
```

Comment:
Right now an expired token leads to a loop where continually reattempt to reimport with the expired token. By deleting it from localstorage I believe it should go into the case on L164 (which returns a console error but not an error toast).